src/c++/perf_analyzer/infer_context.cc

```diff
@@ -236,7 +236,7 @@ void
 InferContext::AsyncCallbackFuncImpl(cb::InferResult* result)
 {
   std::shared_ptr<cb::InferResult> result_ptr(result);
-  bool is_final_response{false};
+  bool is_final_response{true};
```

Comment:
This was causing a bug. When sequence models end early, and the complete ongoing sequences code gets called, the thread_stat_->cb_status_ can have an error, so the default value of is_final_response up here needs to be `true` so that if IsFinalResponse API never gets called, the `total_ongoing_requests_--` code at the bottom still gets run

Reply:
Nice catch. 
Should we try catch this or is setting it to default true seem sane to you?